### PR TITLE
7 get api articles article id comment count

### PR DESCRIPTION
**Pull request by laurie60**

### CODE REVIEW

File: models.js [hunk 1]
```diff
@@ -6,20 +6,31 @@ exports.fetchTopics = () => {
...
-      const article = rows[0];
-      if (!article) {
-        return Promise.reject({
-          status: 404,
-          msg: `No article found with article ID: ${articleId}`,
-        });
-      }
-      return article;
+exports.fetchArticleById = async (articleId) => {
+  const articleArr = await db.query(
```
Comment (karlaferry) [inline models.js]: You are currently doing two queries, but you can definitely only do one.  You just need to query the article and create another column for the comment count. Have a play around :)

**Review by karlaferry — CHANGES_REQUESTED** [review]
Please see my comment and submit again :)

File: models.js (hunk 1 above)
Comment (laurie60) [reply models.js]: Have implemented this on the latest commit

**Review by karlaferry — APPROVED** [review]
Perfect! :)

**Pull request merged**